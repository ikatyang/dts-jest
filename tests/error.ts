declare const error: (v: never) => any;

// @dts-jest
error(1);
