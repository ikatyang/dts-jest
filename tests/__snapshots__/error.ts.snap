// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error(1) 1`] = `"Argument of type '1' is not assignable to parameter of type 'never'."`;
